packages/api-openfin/src/window.js

```diff
@@ -136,27 +138,227 @@ class Window {
     });
   }
 
+  blur() {
+    return new Promise((resolve, reject) => checkWindowOpen(this.innerWindow, () => this.innerWindow.blur(resolve, reject), reject));
+  }
+
   close() {
     return new Promise((resolve, reject) => {
       checkWindowOpen(this.innerWindow, () => this.innerWindow.close(false, resolve, reject), reject);
       this.innerWindow = undefined;
     });
   }
 
+  flashFrame(flag) {
+    if (flag) {
+      return new Promise((resolve, reject) => checkWindowOpen(this.innerWindow, () => this.innerWindow.flash({}, resolve, reject), reject));
+    } else {
+      return new Promise((resolve, reject) => checkWindowOpen(this.innerWindow, () => this.innerWindow.stopFlashing(resolve, reject), reject));
+    }
+  }
+
+  focus() {
+    return new Promise((resolve, reject) => checkWindowOpen(this.innerWindow, () => this.innerWindow.focus(resolve, reject), reject));
+  }
+
+  getBounds() {
+    return new Promise((resolve, reject) => checkWindowOpen(this.innerWindow, () => this.innerWindow.getBounds((bounds) => {
+      resolve({
+        x: bounds.left,
+        y: bounds.top,
+        width: bounds.width,
+        height: bounds.height
+      });
+    }, reject), reject));
+  }
+
+  getChildWindows() {
+    return this.children;
+  }
+
+  getMaximumSize() {
+    return this.getOptions()
+      .then((options) => {
+        return [options.maxWidth, options.maxHeight];
+      });
+  }
+
+  getMinimumSize() {
+    return this.getOptions()
+      .then((options) => {
+        return [options.minWidth, options.minHeight];
+      });
+  }
+
+  getOptions() {
```

Comment:
This isnt implemented in Electron  but is used a lot within the Openfin methods. Is it acceptable to have this method usable by the user?